.github/workflows/main.yml

```
@@ -265,7 +267,8 @@ jobs:
           - { ruby: "3.1", os: "windows-latest", gemfile: "3.1" }
           - { ruby: "3.2", os: "windows-latest", gemfile: "3.2" }
           - { ruby: "3.3", os: "windows-latest", gemfile: "3.3" }
-          - { ruby: "head", os: "windows-latest", gemfile: "3.4" }
+          - { ruby: "3.4", os: "windows-latest", gemfile: "3.4" }
+          - { ruby: "head", os: "windows-latest", gemfile: "3.5" }
```

Just going to turn off the windows 3.4+ for now until ruby installer releases a new version.

```suggestion
          # - { ruby: "3.4", os: "windows-latest", gemfile: "3.4" }
          # - { ruby: "head", os: "windows-latest", gemfile: "3.5" }
```

---

These are available now, you add them back